doc/release-notes/5.0-release-notes.md

```diff
@@ -0,0 +1,321 @@
+
+6. Stop Glassfish; copy domain1 to Payara
+
+   `sudo /usr/local/glassfish4/bin/asadmin stop-domain`
+   `sudo cp -ar /usr/local/glassfish4/glassfish/domains/domain1 /usr/local/payara5/glassfish/domains/`
+
+7. Remove the Glassfish cache directories
+
+   `sudo rm -rf /usr/local/payara5/glassfish/domains/domain1/generated/`  `sudo rm -rf /usr/local/payara5/glassfish/domains/domain1/osgi-cache/`
+
+8. In domain.xml:
+
+ Replace the -XX:PermSize and -XX:MaxPermSize JVM options with -XX:MetaspaceSize and -XX:MaxMetaspaceSize.
+
+   <jvm-options>-XX:MetaspaceSize=256m</jvm-options>
+   <jvm-options>-XX:MaxMetaspaceSize=512m</jvm-options>
+
+Set both Xmx and Xms at startup to avoid runtime re-allocation. Your Xmx value should likely be higher:
+
+   <jvm-options>-Xmx2048m</jvm-options>
+   <jvm-options>-Xms2048m</jvm-options>
+
+Add the below JVM options beneath the -Ddataverse settings:  
+
+   <jvm-options>-Dfish.payara.classloading.delegate=false</jvm-options>
+   <jvm-options>-XX:+UseG1GC</jvm-options>
+   <jvm-options>-XX:+UseStringDeduplication</jvm-options>
+   <jvm-options>-XX:+DisableExplicitGC</jvm-options>
+
+9. Change any full pathnames /usr/local/glassfish4/... to /usr/local/payara5/... or whatever it is in your case. (Specifically check the -Ddataverse.files.directory and -Ddataverse.files.file.directory JVM options)
+
+10. In domain1/config/jhove.conf, change the hard-coded /usr/local/glassfish4 path, as above.
+
+(Optional): If you renamed your service account from glassfish to payara or appserver, update the ownership permissions. The Installation Guide recommends a service account of `dataverse`:
+
+   `sudo chown -R dataverse /usr/local/payara5/glassfish/domains/domain1`
+   `sudo chown -R dataverse /usr/local/payara5/glassfish/lib`
+
+11. You will also need to check that the service account has write permission on the files directory, if they are located outside the old Glassfish domain. And/or make sure the service account has the correct AWS credentials, if you are using S3 for storage.
+
+12. Finally, start Payara:
+
+   `sudo -u dataverse /usr/local/payara5/bin/asadmin start-domain`
+
+13. Deploy the Dataverse 5 warfile:
+
+   `sudo -u dataverse /usr/local/payara5/bin/asadmin deploy /path/to/dataverse-5.0.war`
+
+14. Then restart Payara:
+
+   `sudo -u dataverse /usr/local/payara5/bin/asadmin stop-domain`
+   `sudo -u dataverse /usr/local/payara5/bin/asadmin start-domain`
+
+15. Update Astrophysics Metadata Block (if used)
+
+   `wget https://github.com/IQSS/dataverse/releases/download/5.0/astrophysics.tsv`
+   `curl http://localhost:8080/api/admin/datasetfield/load -X POST --data-binary @astrophysics.tsv -H "Content-type: text/tab-separated-values"`
+
+16. (Recommended) Run ReExportall to update JSON Exports  
+
+   <http://guides.dataverse.org/en/5.0/admin/metadataexport.html?highlight=export#batch-exports-through-the-api>
+
+17. For installations using DataCite, pre-register DOIs:
```

Comment:
Since there is already a detailed section for this step, above - does it need to be here, in the upgrade-to-payara instruction - probably not?

Reply:
We've done this previously, that is, added it in the notes above and called out and individual step.

I added a (Recommended for installations using DataCite) at the start of the instruction. I also added a note for Add doi.dataciterestapiurlstring

Let me know what you think following the updates in https://github.com/IQSS/dataverse/commit/abc4d9bc136d027862dab7b8b72381b89e340d9a